src/libraries/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -133,7 +133,7 @@ private bool RunPredicate(T item)
             }
 
             /// <include file='XmlDocs/CommonXmlDocComments.xml' path='CommonXmlDocComments/Targets/Member[@name="OfferMessage"]/*' />
-            DataflowMessageStatus ITargetBlock<T>.OfferMessage(DataflowMessageHeader messageHeader, T messageValue, ISourceBlock<T> source, bool consumeToAccept)
+            DataflowMessageStatus ITargetBlock<T>.OfferMessage(DataflowMessageHeader messageHeader, T messageValue, ISourceBlock<T>? source, bool consumeToAccept)
```

Comment:
This implementation will throw if `source` is null. Could the interface annotation be updated to take a non-nullable argument?

Reply:
We could.  It would mean adding pragmas to suppress the difference from the interface, and as this is an explicit interface implementation, the annotation won't really affect anyone meaningfully.